app/javascript/components/visualization/controls/ScatterPlotLegend.jsx

```diff
@@ -119,6 +66,7 @@ function LegendEntry({
         className={`scatter-legend-row ${shownClass}`}
         role="button"
         onClick={entryClickFunction}
+        onMouseEnter={() => setActiveTraceLabel(label)}
```

Comment:
Blocker: please log something to Mixpanel for this interaction.

I just recalled from demo last week that there were questions about how discoverable this feature is.  As you noted, logging all of these hover events would be too noisy.   I think you noted we could log, say, only one time when the user first hovers over a legend label -- which would nicely answer questions about discoverability without drastically increasing the number of noisy Mixpanel events.

Reply:
done.  I've just added a simple log for when the mouse enters the legend.  It might be a bit noisier than we need, but shouldn't be too much noisier than our general click-tracking